illumos-utils/src/zfs.rs

```diff
@@ -507,12 +500,22 @@ fn build_zfs_set_key_value_pairs(
 }
 
 /// Describes the ZFS "canmount" options.
+#[derive(Copy, Clone, Debug)]
 pub enum CanMount {
     On,
     Off,
     NoAuto,
 }
 
+impl CanMount {
+    fn wants_mounting(&self) -> bool {
```

Comment:
This is just to make the code more readable below. Should be the same functionality as before.